lib/workers/branch/schedule.js

```diff
@@ -65,14 +77,15 @@ function isScheduledNow(config) {
     logger.warn(errorMessage);
     return true;
   }
+  const [validTimezone, error] = hasValidTimezone(config.timezone);
+  if (!validTimezone) {
+    logger.warn(error);
+    return true;
+  }
   let now = moment();
   logger.debug(`now=${now.format()}`);
-  // Adjust the time if repo is in a different timezone to renovate
-  if (config.timezone) {
```

Comment:
This `if` should be restored because we only want to call this if `config.timezone` is defined

Reply:
restored